application/account-management/Api/ClientTypings/schema.d.ts

```diff
@@ -0,0 +1,172 @@
+        content: {
+          "application/json": components["schemas"]["UpdateTenantCommand"];
+        };
+      };
+      responses: {
+        /** @description Success */
+        200: {
+          content: never;
+        };
+      };
+    };
+    delete: {
+      parameters: {
+        path: {
+          id: string;
+        };
+      };
+      responses: {
+        /** @description Success */
+        200: {
+          content: never;
+        };
+      };
+    };
+  };
+  "/api/tenants": {
+    post: {
+      requestBody: {
+        content: {
+          "application/json": components["schemas"]["CreateTenantCommand"];
+        };
+      };
+      responses: {
+        /** @description Success */
+        200: {
+          content: never;
+        };
+      };
+    };
+  };
+  "/api/users/{id}": {
+    get: {
+      parameters: {
+        path: {
+          id: string;
+        };
+      };
+      responses: {
+        /** @description Success */
+        200: {
+          content: never;
+        };
+      };
+    };
+    put: {
+      parameters: {
+        path: {
+          id: string;
+        };
+      };
+      requestBody: {
+        content: {
+          "application/json": components["schemas"]["UpdateUserCommand"];
+        };
+      };
+      responses: {
+        /** @description Success */
+        200: {
+          content: never;
+        };
+      };
+    };
+    delete: {
+      parameters: {
+        path: {
+          id: string;
+        };
+      };
+      responses: {
+        /** @description Success */
+        200: {
+          content: never;
+        };
+      };
+    };
+  };
+  "/api/users": {
+    post: {
+      requestBody: {
+        content: {
+          "application/json": components["schemas"]["CreateUserCommand"];
+        };
+      };
+      responses: {
+        /** @description Success */
+        200: {
+          content: never;
+        };
+      };
+    };
+  };
+}
+
+export type webhooks = Record<string, never>;
+
+export interface components {
+  schemas: {
+    CreateTenantCommand: {
+      subdomain?: string | null;
+      name?: string | null;
+      phone?: string | null;
+      email?: string | null;
+    };
+    CreateUserCommand: {
+      tenantId?: components["schemas"]["TenantId"];
+      email?: string | null;
+      userRole?: components["schemas"]["UserRole"];
+    };
+    TenantId: {
+      value?: string | null;
+    };
+    UpdateTenantCommand: {
+      name?: string | null;
+      phone?: string | null;
+    };
+    UpdateUserCommand: {
+      userRole?: components["schemas"]["UserRole"];
+      email?: string | null;
+    };
+    /** @enum {string} */
+    UserRole: "TenantUser" | "TenantAdmin" | "TenantOwner";
+  };
+  responses: never;
+  parameters: never;
+  requestBodies: never;
+  headers: never;
+  pathItems: never;
+}
+
+export type $defs = Record<string, never>;
+
+export type external = Record<string, never>;
+
+export type operations = Record<string, never>;
```

Comment:
this file is autogenerated - there are pros/cons for checking it in / leaving it out - might be nice for debugging changes